runelite-client/src/main/java/net/runelite/client/plugins/combat/CombatPlugin.java

```diff
@@ -22,51 +22,81 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package net.runelite.client.plugins.combatlevel;
+package net.runelite.client.plugins.combat;
 
 import com.google.common.eventbus.Subscribe;
+import com.google.inject.Provides;
 import java.text.DecimalFormat;
 import javax.inject.Inject;
 import net.runelite.api.Client;
 import net.runelite.api.Experience;
 import net.runelite.api.GameState;
 import net.runelite.api.Skill;
+import net.runelite.api.events.ConfigChanged;
 import net.runelite.api.events.GameTick;
 import net.runelite.api.widgets.Widget;
 import net.runelite.api.widgets.WidgetInfo;
+import net.runelite.client.config.ConfigManager;
 import net.runelite.client.plugins.Plugin;
 import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.ui.overlay.Overlay;
 
 @PluginDescriptor(
-	name = "Combat Level"
+	name = "Combat"
 )
-public class CombatLevelPlugin extends Plugin
+public class CombatPlugin extends Plugin
 {
-	private final DecimalFormat decimalFormat = new DecimalFormat("#.###");
+	private static final DecimalFormat DECIMAL_FORMAT = new DecimalFormat("#.###");
+	private static final int FIXED_VIEWPORT_MULTI_Y = 300;
 
 	@Inject
-	Client client;
+	private CombatOverlay overlay;
+
+	@Inject
+	private Client client;
+
+	@Inject
+	private CombatConfig config;
+
+	@Provides
+	CombatConfig getConfig(ConfigManager configManager)
+	{
+		return configManager.getConfig(CombatConfig.class);
+	}
+
+	@Override
+	public Overlay getOverlay()
+	{
+		return overlay;
+	}
 
 	@Override
 	protected void shutDown() throws Exception
 	{
-		Widget combatLevelWidget = client.getWidget(WidgetInfo.COMBAT_LEVEL);
+		restoreCombatLevel();
+	}
 
-		if (combatLevelWidget != null)
+	@Subscribe
+	public void onConfigChanged(ConfigChanged configChanged)
+	{
+		if (configChanged.getGroup().equals("combat")
+			|| !config.showCombatDecimals())
 		{
-			String widgetText = combatLevelWidget.getText();
-
-			if (widgetText.contains("."))
-			{
-				combatLevelWidget.setText(widgetText.substring(0, widgetText.indexOf(".")));
-			}
+			restoreCombatLevel();
 		}
 	}
 
 	@Subscribe
 	public void updateCombatLevel(GameTick event)
 	{
-		if (client.getGameState() != GameState.LOGGED_IN)
+		Widget multiWidget = client.getWidget(WidgetInfo.FIXED_VIEWPORT_MULTI_INDICATOR);
+		if (multiWidget != null)
+		{
+			multiWidget.setRelativeY(FIXED_VIEWPORT_MULTI_Y);
```

Comment:
not sure about permanently adjusting this.. maybe only if the level range is showing?